scripts/update

```diff
@@ -50,7 +50,7 @@ then
         echo "Updating Scala dependencies"
         docker-compose up -d api-server
 
-        docker-compose exec api-server ./sbt update
+        docker-compose exec -T api-server ./sbt update
```

Comment:
I'm fine with this fix, but also couldn't we just use `run`? Because of the volume mounts, the results of `update` will be persisted, right?

Reply:
That would definitely work. I only used `exec` here because I knew the container would be running.